src/js/angular/import/controllers.js

```diff
@@ -31,16 +32,11 @@ const modules = [
 
 const importViewModule = angular.module('graphdb.framework.impex.import.controllers', modules);
 
-const OPERATION = {
+export const OPERATION = {
```

Comment:
Why this is exported? As far as I can see, it is used only here?

Reply:
I forgot to remove this constant. The method where it is used has to use 'TABS'. This is the reason the tests failed.

Reply:
Fixed.